chart/values.schema.json

```diff
@@ -7169,10 +7164,10 @@
                             "type": "string",
                             "default": "tests/dags"
                         },
-                        "wait": {
+                        "period": {
                             "description": "Interval between git sync attempts in seconds. High values are more likely to cause DAGs to become out of sync between different components. Low values cause more traffic to the remote git repository.",
-                            "type": "integer",
-                            "default": 5
+                            "type": "string",
+                            "default": "5s"
```

Comment:
This change concerns me though. Maybe we can continue to use the wait env var so we can use the float seconds path instead?